browser/brave_browser_process_impl.cc

```diff
@@ -155,6 +156,15 @@ BraveBrowserProcessImpl::referrer_whitelist_service() {
   return referrer_whitelist_service_.get();
 }
 
+brave_shields::SiteSpecificScriptService*
```

Comment:
I'm not sure if it makes sense to have a new global service for this. Since all the settings are per-profile, wouldn't a keyed service make more sense?

Reply:
@bridiver maybe that comment is more for every one of these DAT file wrapped component services and not just this one in particular?